### PR TITLE
Deploy baremetal bootstrap services with MCO

### PR DESCRIPTION
**Pull request by bcrochet**
Currently we have two very different sets of manifests and templates
for deploying the baremetal infrastructure services. One lives in the
installer, the other lives in MCO. This change removes the services
from the installer and makes use of the files being added to the MCO
that serve the same purpose but keep all of the templates in the same
repo and allow us to use the same MCO data for populating dynamic
values.

On platforms that don't need these services on the bootstrap node,
MCO will not render these files. We should check for the existence
of the relevant manifest and if present try to copy those files.

The static-pod-resources are grouped with the manifest because the
manifests require them, so it would be an error for one to exist
without the other.

### CODE REVIEW

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 1]
```diff
@@ -211,13 +211,17 @@ then
...
+	mkdir --parents /etc/mcc/bootstrap /etc/mcs/bootstrap /etc/kubernetes/manifests /etc/kubernetes/static-pod-resources
 	cp mco-bootstrap/bootstrap/manifests/* /etc/mcc/bootstrap/
 	cp openshift/* /etc/mcc/bootstrap/
 	# 4. read ImageContentSourcePolicy objects generated by the installer
 	cp manifests/* /etc/mcc/bootstrap/
 	cp auth/kubeconfig-kubelet /etc/mcs/kubeconfig
 	cp mco-bootstrap/bootstrap/machineconfigoperator-bootstrap-pod.yaml /etc/kubernetes/manifests/
+        if [ -d mco-bootstrap/baremetal/manifests ]; then
+            cp mco-bootstrap/baremetal/manifests/* /etc/kubernetes/manifests/
+            cp -r mco-bootstrap/baremetal/static-pod-resources/* /etc/kubernetes/static-pod-resources/
```
Comment (abhinavdahiya) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template]: nit: i think this script uses full names for flags `--recursive`

**Comment by metal3ci** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/970/

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template (hunk 1 above)
Comment (abhinavdahiya) [reply data/data/bootstrap/files/usr/local/bin/bootkube.sh.template]: > https://github.com/openshift/installer/pull/2119#issuecomment-516652885

@stbenjam will you be filing follow-ups to fix this..

Comment (stbenjam) [reply data/data/bootstrap/files/usr/local/bin/bootkube.sh.template]: The other well known shell commands (`sed -i -e`, `rm -rf`) do not use long flags in this script. If you really want it this way, this  fixes them all: https://github.com/openshift/installer/pull/2130

**Pull request merged**